src/query.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const protons = require('protons')
+const schema = `
+message Query {
```

Comment:
Just wondering why there's a query, queryresponse.

We're not really querying that way. We're just brodcasting.

Also is the query id being used?

Over pubsub we can't do request/response anyway

Reply:
The major downside to the interval is that there will be a delay in us learning about all of the peers. If the interval is say, 10seconds, and we join at the start of a new interval, it will take 10 seconds for us to discover all peers. This is a pretty minor downside though, especially since we are working on persistent peer store support.